transports/sftp/src/test/resources/mule-sftp-auto-delete-implicit-connector.xml

```diff
@@ -0,0 +1,27 @@
+<mule xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
this file is almost identical to the other one. Use a spring profile or at the very least an import